website/src/_posts/2019-05-22-beam-2.13.0.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
Looks like this just repeats the notes for 2.12.0, so we need to collect recent updates from the community and update this.

My two cents (cc: @pabloem, @chamikaramj for cross-check):


```
### I/Os
BigQuery IO now supports BYTES datatype on Python 3. 
Avro IO support enabled on Python 3.
For Python 3 pipelines, the default Avro library used by Beam AvroIO and Dataflow workers was switched from avro-python3 to fastavro.  
```

```
### Breaking Changes
* When writing BYTES Datatype into Bigquery with Beam Bigquery IO on Python DirectRunner, users need to base64-encode bytes values before passing them to Bigquery IO. Accordingly, when reading bytes data from BigQuery, the IO will also return base64-encoded bytes. This change only affects BigQuery IO on Python DirectRunner. New DirectRunner behavior is consistent with treatment of Bytes by Beam Java Bigquery IO, and Python Dataflow Runner.
```

Reply:
LGTM on these updates. Thanks.

Reply:
+1

Reply:
Added.